week2/prep-exercises/1-generate-random-person-details/README.md

```diff
@@ -8,7 +8,7 @@ A common problem is to know where to start!
 
 ## 1. Setup
 
-1. First, let's make sure we have some place to code. Run `create-react-app` in this folder to create your environment
+1. First, let's make sure we have some place to code. Run `npm create vite assignment -- --template react` in this folder to create your environment
```

Comment:
Is the `--` in the middle required here? Or was it a type?

Reply:
It is explicitly mentioned in the [Getting Started](https://vitejs.dev/guide/#scaffolding-your-first-vite-project)